sufia-models/app/models/concerns/sufia/generic_file/featured.rb

```diff
@@ -1,11 +0,0 @@
-module Sufia
```

Comment:
@grosscol I'm thinking we may want to add a module to GenericWork so it knows if it is featured.  This is not required for this ticket, but will be required for #1049.  This does not block this PR from being merged.